test/api/assignments_test.rb

```diff
@@ -76,7 +76,7 @@ def test_api_accepts_submission_on_completed_exercise
       Xapi.stub(:exists?, true) do
         solution = { "THE CODE" => "go/one/code.go" }
         post '/user/assignments', { key: alice.key, code: 'THE CODE', path: 'go/one/code.go', 
-          'language' => 'go', 'problem' => 'one', solution: solution }.to_json
+        'language' => 'go', 'problem' => 'one', solution: solution }.to_json
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.